server/controllers/profile.js

```diff
@@ -0,0 +1,123 @@
+const Profile = require("../models/profile");
+const asyncHandler = require("express-async-handler");
+const profile = require("../models/profile");
```

Comment:
```suggestion
```
Let's remove this import as we are already importing the Profile model on `line 1`

Reply:
Removed the extra line